contents/data_vis_makie.md

```diff
@@ -48,3 +48,6 @@ save("filename.pdf", fig; pt_per_unit=0.5)
 
 For `png`'s you specify `px_per_unit`.
 See [Backends & Output](https://makie.juliaplots.org/stable/documentation/backends_and_output/) for details.
+
+Another important issue is to actually visualize your output plot. Note that for `CairoMakie.jl` the Julia REPL is not able to show plots, so you will need and IDE (Integrated Development Environment) such as VSCode, Jupyter or Pluto that supports `png` or `svg` as output.
```

Comment:
```suggestion
Another important issue is to actually visualize your output plot.
Note that for `CairoMakie.jl` the Julia REPL is not able to show plots, so you will need and IDE (Integrated Development Environment) such as VSCode, Jupyter or Pluto that supports `png` or `svg` as output.
```
Consistency :)

Reply:
you mean -so you will need `an` IDE-, could you please update.